### PR TITLE
ember-cli notes 20 july 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: ember-cli/2017-07/july-20.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# ember-cli Meeting Notes - 2017-07-20
+
+## Attendees
+
+
+
+## Agenda
```
Comment (hjdivad) [inline ember-cli/2017-07/july-20.md]: Reply here to add agenda items

Comment (kratiahuja) [inline ember-cli/2017-07/july-20.md]: FCP: https://github.com/ember-cli/rfcs/pull/108 ? (Chad ok'd it)

**Comment by hjdivad** [comment]
cc @trabus @kellyselden @kratiahuja @mbutki

**Comment by hjdivad** [comment]
also @rwjblue any chance we can merge some of the older ones that people are 👍  on?

**Pull request merged**